frontend/src/metabase/embedding-sdk/theme/default-component-theme.ts

```diff
@@ -110,7 +110,6 @@ export const DEFAULT_EMBEDDED_COMPONENT_THEME: MetabaseComponentTheme = merge<
   table: {
     cell: {
       fontSize: FONT_SIZES.tableCell.em,
-      backgroundColor: "white",
```

Comment:
This is why it was falling back to `white`, as it was applying white background to all tables, regardless of whether the page is a dark-themed page. We cannot set `var(--mb-color-background)` or `bg-white` here as the BaseCell component expects a hex value, so it's better to use a dynamic default.